src/broker_statement/ib/corporate_actions.rs

```diff
@@ -22,7 +25,20 @@ impl RecordParser for CorporateActionsParser {
     }
 
     fn parse(&mut self, _parser: &mut StatementParser, record: &Record) -> EmptyResult {
-        self.corporate_actions.push(parse(record)?);
+        let action = parse(record)?;
+        match action.action {
+            CorporateActionType::Liquidation {
+                quantity,
+                price,
+                volume,
+                commission,
+            } => {
+                self.stock_sells.push(StockSell::new_trade(
```

Comment:
Why you decided to do it here and introduce `self.stock_sells` instead of just do the same in the simpler way at https://github.com/KonishchevDmitry/investments/pull/70/files#diff-3e1980fff2025a6c1bcdd3e062ee80131e9e35330b0b200d7ac6a682d5a49b42R251? Is it for purpose to workaround something or it's just a first way you found to implement the feature and I can safely refactor it?

Reply:
It was the first thing that came to my mind. So, yeah, it's far from optimal